pkg/util/iswimi/iswimi_test.go

```diff
@@ -0,0 +1,45 @@
+package iswimi
+
+// Copyright (c) Microsoft Corporation.
+// Licensed under the Apache License 2.0.
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/Azure/ARO-RP/pkg/api"
+)
+
+func TestIswimi(t *testing.T) {
+	tests := []struct {
+		name    string
+		cluster api.OpenShiftClusterProperties
+		want    bool
+	}{
+		{
+			name: "Cluster is Workload Identity",
+			cluster: api.OpenShiftClusterProperties{
+				PlatformWorkloadIdentityProfile: &api.PlatformWorkloadIdentityProfile{},
+				ServicePrincipalProfile:         nil,
+			},
+			want: true,
+		},
+		{
+			name: "Cluster is Classic",
```

Comment:
```suggestion
			name: "Cluster is Service Principal",
```

I'm suggesting this change for the same reason as explained in my doc comment on the `isWimi()` function.